pkg/cmd/admin/policy/who_can.go

```diff
@@ -68,18 +68,19 @@ func (o *whoCanOptions) complete(args []string) error {
 }
 
 func (o *whoCanOptions) run() error {
-	resourceAccessReview := &authorizationapi.ResourceAccessReview{}
-	resourceAccessReview.Resource = o.resource
-	resourceAccessReview.Verb = o.verb
+	authorizationAttributes := authorizationapi.AuthorizationAttributes{
+		Resource: o.resource,
+		Verb:     o.verb,
```

Comment:
why not just set `Namespace` to `bindingNamespace` when `!allNamespaces`, and use `o.client.ResourceAccessReviews()` in all cases. I thought LocalResourceAccessReviews were for backwards compatibility only, not the preferred mechanism